www/src/components/pdf/PDFViewer.tsx

```diff
@@ -0,0 +1,31 @@
+import { Grid } from '@material-ui/core';
+import React, { FC } from 'react';
+import { useEffect } from 'react';
+
+import ViewSDKClient from './ViewerSDKClient';
+
+// TODO: Modify to properly pass necessary props to make get resume from api
+type PDFViewerProps = {
+    id: string;
+};
+
+const PDFViewer: FC<PDFViewerProps> = () => {
+    useEffect(() => {
+        const viewSDKClient = new ViewSDKClient();
+        viewSDKClient.ready().then(() => {
+            /* Invoke file preview */
+            viewSDKClient.previewFile('adobe-dc-view', {
+                /* Control the viewer customization. */
+                showAnnotationTools: true,
+                enableFormFilling: true,
+                showLeftHandPanel: false,
```

Comment:
What is in the left hand panel?

Reply:
When you open adobe pdf viewer it's like the veritcal bar that shows all the pages in the pdf. I figured it's not necessary cus resumes should just be 1 page

Reply:
Will scrolling work for 2 page resumes?

Reply:
Yeah, just tested with a 2-page pdf, you'll be able to scroll (though there shouldn't be any 2 page resumes)

Reply:
> though there shouldn't be any 2 page resumes

I agree. But you would be surprised how many people have 2 page resumes.